components/Header/Header.tsx

```diff
@@ -60,6 +60,7 @@ export const Header: React.FC<HeaderProps> = (props) => {
       <Sticky>
         <BottomHeader>
           <MainMenu
+            showMenuHeader={true}
```

Comment:
For true boolean props we can just pass the prop name without assignment

For false boolean props we can just leave out the propname entirely unless the default prop value is true

So here we can change this to 

<MainMenu showMenuHeader ...